README.md

```diff
@@ -45,6 +45,19 @@ Argo is a [Cloud Native Computing Foundation (CNCF)](https://cncf.io/) hosted pr
 
 ![Screenshot](docs/assets/screenshot.png)
 
+## Try Argo using Nix
+Nix is a package manager / build tool which focusses on reproducible build environments.
+Argo Workflows has some basic support for Nix which is enough to get Argo Workflows up and running with minimal effort. 
+Here are the steps to follow: 
+  1. modify the hosts file according to [this](https://argoproj.github.io/argo-workflows/running-locally/), don't worry about the other instructions. 
+  2. setup a k8s cluster, k3d is the recommended solution here. 
+  3. Install [Nix](https://nixos.org/download.html) 
+  4. Run "nix develop" (you will have to use the experimental feature flags on a fresh install, the features you need are "nix-command" and "flakes").
```

Comment:
nit - it'd be better to type the full command here: `nix develop --extra-experimental-features nix-command --extra-experimental-features flakes`